agreements/bootstrapping/2024-05-06-bootstrapping-objenesis.md

```diff
@@ -0,0 +1,79 @@
+---
+status: draft
```

Comment:
Oversight on my part. 🙄 Can you remove this 3 line draft indicator?

Reply:
Sure. Done